compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_control_flow.rs

```diff
@@ -246,19 +246,20 @@ impl<F: AcirField + DebugToString, Registers: RegisterAllocator> BrilligContext<
         });
     }
 
-    /// Emits brillig bytecode to jump to a trap condition if `condition`
-    /// is false.
+    /// Emits brillig bytecode to jump to a trap condition if `condition` is false,
```

Comment:
nit: maybe we remove "Emits brillig bytecode" as we that is implied by the `codegen_` prefix.